cmd/minishift/cmd/oc_env.go

```diff
@@ -65,11 +65,11 @@ func executeOcTemplateStdout(shellCfg *OcShellConfig) error {
 
 var ocEnvCmd = &cobra.Command{
 	Use:   "oc-env",
-	Short: "Sets path of 'oc' binary.",
-	Long:  `Sets path of OpenShift client binary, 'oc'.`,
+	Short: "Sets the path of the 'oc' binary.",
+	Long:  `Sets the path of OpenShift client binary, or 'oc'.`,
```

Comment:
that's the same thing but I can think of another way of indicating this that doesn't involve the word "or" if it bothers you.